2-mandatory/2-financial-times.js

```diff
@@ -14,7 +16,7 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
```

Comment:
This is a very good attempt - but there's one small problem.
Your code returns the title with the fewest characters... But for this exercise, we would like the title with the fewest words 😄 
Can you think of how you can change your code to achieve this?